client/signup/steps/site-info-collection/index.tsx

```diff
@@ -0,0 +1,277 @@
+import { useTranslate } from 'i18n-calypso';
+import { ChangeEvent, useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { saveSignupStep } from 'calypso/state/signup/progress/actions';
+import {
+	updateSiteInfoCurrentIndex,
+	updateSiteInfoValues,
+} from 'calypso/state/signup/steps/site-info-collection/actions';
+import { SiteInfoCollectionData } from 'calypso/state/signup/steps/site-info-collection/schema';
+import {
+	getSiteInfoCollectionData,
+	getSiteInfoCollectionCurrentIndex,
+} from 'calypso/state/signup/steps/site-info-collection/selectors';
+import { ContactInformation, TextInputField, SocialMediaProfiles } from './form-components';
+import SectionWrapper from './section-wrapper';
+import { SectionProps, ValidationErrors, ValidatorFunction } from './types';
+
+import './style.scss';
+
+interface SiteInformationCollectionProps {
+	additionalStepData: object;
+	stepName: string;
+	submitSignupStep: ( step: { stepName: string }, formValues: SiteInfoCollectionData ) => void;
+	goToNextStep: () => void;
+}
```

Comment:
I can distinctly remember a convention where all types are encouraged to moved to the common file. I myself found it useful since it tends to reduce clutter in the file with logiv and the types file acts as sort of a central point for documentation to given section.

But no big issue here :)
Probably @tyxla or @jsnajdr can weigh in on this in case I am on the wrong page here.

Reply:
We don't have an official convention yet, so feel free to use either a single types file or spread them among multiple - whatever makes the most sense for your project at this time.